content/PhoneList.qml

```diff
@@ -1,5 +1,83 @@
 import QtQuick 2.15
 import QtQuick.Controls 2.15
+import QtQuick.Studio.Components 1.0
+import QtQuick.Layouts
+import Qt5Compat.GraphicalEffects
+
+Item {
+    id: main
+    width: 860
+    height: 540
+
+    PhoneListToolBar {
+    }
+
+    Component {
+            id: contactDelegate
+
+            Rectangle {
+                id: wrapper
+                width: parent.width
+                height: 150
+
+                Button {
+                    id: button
+                    width: parent.width
+                    anchors.fill: wrapper
+
+                    Row {
+                        anchors.verticalCenter: parent.verticalCenter
+                        anchors.leftMargin: 50
+                        Rectangle {
+                            id: profilePhotoSegment
+                            x: 100

```

Comment:
Don't use 'x' property in Row

Reply:
You should use property Row such as `leftPadding` and `spacing`, and width for sub-items. If you prefer to use "x" then using "Row" makes no sense because "Row" sets the x-position sub-items itself. You can then use basic "Item" instead of Row.